src/common/migrations.js

```diff
@@ -544,6 +544,16 @@ const fixDropDownNumberValues = (data) => {
     return data;
 };
 
+const updateShiftNode = (data) => {
```

Comment:
Why did you add a migration? Added a new dropdown (or in general, any input with a default value) shouldn't require one.